Bucketeer/Sources/Internal/Evaluation/EvaluationStorageImpl.swift

```diff
@@ -65,36 +65,36 @@ final class EvaluationStorageImpl: EvaluationStorage {
             try evaluationDao.deleteAll(userId: userId)
             try evaluationDao.put(userId: userId, evaluations: evaluations)
         }
-        // Update cache directly after we have force update
+        // Update cache directly
         evaluationMemCacheDao.set(key: userId, value: evaluations)
         self.evaluatedAt = evaluatedAt
     }
 
     func update(evaluations: [Evaluation], archivedFeatureIds: [String], evaluatedAt: String) throws -> Bool {
-        var activeEvaluations = [Evaluation]()
-        var archivedEvaluationIds = [String]()
-        try evaluationDao.startTransaction {
-            // 1. Update evaluation with new data
-            try evaluationDao.put(userId: userId, evaluations: evaluations)
-            // 2. Get current data in db
-            let evaluationsInDb = try evaluationDao.get(userId: userId)
-            // 3. Filter active & archived
-            for evaluation in evaluationsInDb {
-                if archivedFeatureIds.contains(evaluation.featureId) {
-                    archivedEvaluationIds.append(evaluation.id)
-                } else {
-                    activeEvaluations.append(evaluation)
-                }
-            }
-            // 4. Remove all the evaluations which have the same id in the list archivedEvaluationIds
-            if (archivedEvaluationIds.count > 0) {
-                try evaluationDao.deleteByIds(archivedEvaluationIds)
+        // 1. Get current data in db
+        var activeEvaluationsDict = try evaluationDao.get(userId: userId)
```

Comment:
Please rename it to `currentEvaluations`.

Reply:
changed in https://github.com/bucketeer-io/ios-client-sdk/pull/44/commits/61d8b80324fff4d86a32292ef5c5f84c357dd79c